app/views/jobseekers/job_applications/build/declarations.html.slim

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title_prefix, t(".title")
+
+= render(Jobseekers::JobApplicationHeadingComponent.new(vacancy: @job_application.vacancy))
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    .govuk-caption-m Step #{@current_step_number} of 9
+    h1.govuk-heading-l = t(".title")
+
+    = form_for @form, url: wizard_path, method: :patch do |f|
+      = f.govuk_error_summary
+
+      = f.govuk_collection_radio_buttons :banned, %w[yes no], :to_s, :capitalize, inline: true
+
+      = f.govuk_radio_buttons_fieldset :close_relationships do
+        = f.govuk_radio_button :close_relationships, :yes, link_errors: true do
+          = f.govuk_text_field :close_relationships_details, label: { size: "s" }
```

Comment:
The design for the QTS conditional question had this not in bold (i.e omit `label: { size: "s" }`), so I imagine they should be consistent?